site-src/contributing/gep.md

```diff
@@ -0,0 +1,107 @@
+# Gateway Enhancement Proposal (GEP)
+
+Gateway Enhancement Proposals (GEPs) will serve a similar purpose to
+the [KEP][kep] process for the main Kubernetes project:
+
+1. Ensure that changes to the API follow a known process and discussion
+  in the OSS community.
+1. Make changes and proposals discoverable (current and future).
+1. Document design ideas, tradeoffs, decisions that were made for
+  historical reference.
+
+As this is still a relatively new project, we don't wish to encumber
+proposals with a large amount of boilerplate process. The most
+important part of the GEP are the last two items: making sure that we
+can easily find previous discussions and alternatives considered.
+
+## Process
+
+### New GEP
+
+* File an issue that will be used to track the GEP progress. Use the
+  existing issue templates.
+* Discussion and comments occur on the issue and shared docs. Any
+  relevant shared documents should be linked off of the Github issue.
+  * Create a GEP document in `site-srcs/gep/gep-<issue
+    #>-short-description.md` with status `proposed`; the sections that
+    MUST be kept up to date are `TLDR` and `References`. Other
+    sections can refer to the issue or shared doc temporarily.
+* When there is consensus on the approach, Consolidate any content,
+  links into the document (see recommended format below). Submit the
+  PR for merging.
+* Propagate iterations on the GEP proposal from the issue/shared docs
+  to the GEP.
+* When there are no further comments, update GEP with status
+  "accepted". This means that we are committing to implementing/have
+  implemented the feature.
+* If we find that the proposal is not going to move forward, mark the
+  status as `archived`.
+* Tracking issue should be marked as closed.
+
+## Format
+
+
+``` markdown
+# GEP-1234: Add Chaosnet protocol support
```

Comment:
I think it would be ideal to make this a template file that is in the GEPs directory.

Reply:
+1. Having an md file in a geps directory would be helpful.

I think we should also create an issue template of type enhancement to further ease the process.

Reply:
I will add the template.